pages/common/git-archive-file.md

```diff
@@ -0,0 +1,9 @@
+# git archive-file
+
+> Copy the current Git branch into a zip archive.
```

Comment:
thoughts?
```suggestion
> Export all the files of the current Git branch into a zip archive.
```

Reply:
Seems ok, however copy is easier to read

Reply:
'Tis, but export is more descriptive ;)

Reply:
@marchersimon thoughts?

Reply:
`Export` is better, also I think it should say `current Git commit` @bl-ue

Reply:
```suggestion
> Export all the files of the current Git commit into a zip archive.
```

Reply:
Couldn't we just leave out `all the files of`, so we just have:
`Export the current Git commit into a zup archive` ?

Reply:
I personally say no, since to me, exporting a commit would be just taking the most recent commits changes.

Reply:
I agree with you @navarroaxel.  @CleanMachine1 branches in Git are really simply pointers to commits, and nothing else.  You can also be checked out on a commit, `e.g. git checkout a1b2c3d`, which is called _detached HEAD._  [This StackOverflow article](https://stackoverflow.blog/2021/04/05/a-look-under-the-hood-how-branches-work-in-git/) I found very explanatory.

Reply:
I was thinking of `all the files of` because, as @CleanMachine1 pointed out, it may not be obvious to a new user that it's exporting the entire tree.

Reply:
Can I have a suggestion to add, or tell me which

Reply:
I think @bl-ue's original suggestion is fine.